src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -393,10 +396,15 @@ CBMC_TIMEOUT ?= 21600
 # Replace all uses of char * by a struct that carries that string,
 # and also the underlying allocation and the C string length.
 STRING_ABSTRACTION ?=
+ifdef STRING_ABSTRACTION
```

Comment:
🥝 Why do we need both `ifdef` and `ifneq`? I think `ifneq` is sufficient and consistent with what we do in the rest of the file

Reply:
I was following the same pattern used for other variables. I thought the idea was to avoid the comparing and `strip` at all if the variable is not even defined.

Reply:
For example,
```Makefile
ifdef RESTRICT_FUNCTION_POINTER
  ifneq ($(strip $(RESTRICT_FUNCTION_POINTER)),)
    CBMC_RESTRICT_FUNCTION_POINTER := $(patsubst %,--restrict-function-pointer %, $(RESTRICT_FUNCTION_POINTER))
  endif
endif
```